edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/TestServer/Startup.cs

```diff
@@ -23,8 +23,6 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IController, EdgeletTestImplementation>();
         }
 
-        // TODO: Remove warning disable for Obsolete when project is moved to dotnetcore 3.0
-#pragma warning disable 612, 618
```

Comment:
Removed and built locally, and didn't see any warnings; therefore took it out.

Reply:
Does it build with netcore 2.1 and netcore 3.0?

Reply:
reverted